specs/typography.md

```diff
@@ -0,0 +1,177 @@
+| ```<h3>``` | 1.17em   | 18.7px |
+| ```<h4>``` | 1em      | 16px   |
+| ```<h5>``` | 0.83em   | 13.2px |
+| ```<h6>``` | 0.67em   | 10.7px |
+
+Any of these elements may be used with FAST-DNA styles applied.
+
+### Use Cases
+
+- Defining page hierarchy through one or more sequences of ```<h1>```, ```<h2>```, ```<h3>```, etc.
+- Associating a ```<label>``` with an input field
+- Styling text as secondary or metatext
+- Adding a caption to an image or figure
+
+### Features
+
+- An extensible type ramp based on a default font size
+- A set of semantically named styles based on the underlying type ramp
+- A default mapping for style to standard elements
+
+### Risks and Challenges
+
+It's possible that styles only (not components) will be harder to implement consistently. We may want to consider simple custom elements that consistently apply the same underlying styles as well as any necessary aria tags.
+
+### Prior Art/Examples
+
+- FAST-DNA (React): [Heading](https://explore.fast.design/components/heading), [Subeading](https://explore.fast.design/components/subheading), [Paragraph](https://explore.fast.design/components/paragraph), [Label](https://explore.fast.design/components/label), [Caption](https://explore.fast.design/components/caption), [Metatext](https://explore.fast.design/components/metatext), [Typography](https://explore.fast.design/components/typography)
+- [Material Design](https://material.io/design/typography/the-type-system.html)
+- [Bootstrap](https://getbootstrap.com/docs/4.4/content/typography)
+- [Ant Design](https://ant.design/components/typography)
+- [Semantic UI](https://semantic-ui.com/elements/header.html)
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/header/definition)
+
+Most other component frameworks either maintain the base font size or set it smaller to 14px. They also typically maintain heading element sizes based on `em` or `rem`, though some use individual specific values. Typically heading sizes are close to the browser default, though Material Design goes a lot larger at the top.
+
+---
+
+## Design
+
+The type styles published are consistently named so they are easy to identify and consume. A clear separation is made between the semantic naming and the raw type ramp.
+
+Default styles are applied to standard elements such that headings and other base typography pick up the adaptive styling without needing to add class names.
+
+(**Editorial**: can we do this, or do we need to build this support?)
+
+### ~~API~~
+
+*~~The key elements of the component's public API surface~~*
+
+### Adaptive Styling Properties (**Editorial**: Added this section)
+
+- `type-ramp-base-size` = "14px"  
+The base size for primary body text. Other type ramp sizes may eventually be determined from this starting value.
+
+### Anatomy and Appearance
+
+The type ramp is constructed as css variables that go plus and minus from the base. This allows a consumer to extend the ramp if necessary.
```

Comment:
This is a deviation from what we have today, where the ramp simply extends 1-9. I'd like to understand the benefits this approach provides - can you expand on the *why* of this +- scale?

Reply:
Cool thanks for explaining - I think that makes sense.